otherlibs/dune-rpc/private/exported_types.ml

```diff
@@ -189,17 +585,18 @@ module Diagnostic = struct
 
   module Related = struct
     type t =
-      { message : unit Pp.t
+      { message : User_message.Style.t Pp.t
       ; loc : Loc.t
       }
 
-    let message t = t.message
+    let message t = t.message |> Pp.map_tags ~f:(fun _ -> ())
+    let message_with_style t = t.message
```

Comment:
We add an extra function allowing for an opt in to the stylised message. This keeps our API version compatible. The old version is just the new version with the style tags removed.